test/modules/command/dc/dc-offer-create-database-command.test.js

```diff
@@ -0,0 +1,135 @@
+const {
+    describe, before, after, it,
+} = require('mocha');
+const { assert } = require('chai');
+const DCOfferCreateDatabaseCommand = require('../.././../../modules/command/dc/dc-offer-create-database-command');
+const models = require('../../../../models/index');
+const SystemStorage = require('../../../../modules/Database/SystemStorage');
+const BN = require('bn.js');
+const sleep = require('sleep-async')().Promise;
+const Utilities = require('../.././../../modules/Utilities');
+const GraphStorage = require('../.././../../modules/Database/GraphStorage');
+const Storage = require('../.././../../modules/Storage');
+const { Database } = require('arangojs');
+const CommandResolver = require('../.././../../modules/command/command-resolver');
+const CommandExecutor = require('../.././../../modules/command/command-executor');
+const awilix = require('awilix');
+
+function buildSelectedDatabaseParam(databaseName) {
+    return {
+        username: process.env.DB_USERNAME,
+        password: process.env.DB_PASSWORD,
+        database: databaseName,
+        host: process.env.DB_HOST,
+        port: process.env.DB_PORT,
+        database_system: 'arango_db',
+    };
+}
+
+describe('Checks DCOfferCreateDatabaseCommand', function () {
+    this.timeout(5000);
+    let graphStorage;
+    let systemDb;
+    let myConfig;
+    let myGraphStorage;
+    let container;
+    let myCommand;
+
+    const databaseName = 'dc_offer_create_db';
+
+    before('Setup preconditions and call DCOfferCreateDatabaseCommand execute function', async () => {
+        try {
+            await SystemStorage.connect();
+        } catch (error) {
+            console.log('Smth went wrong with SystemStorage.connect()');
```

Comment:
can we use logger instead of `console.log(...)`?

Reply:
agreed, will be done